python-sdk/src/astro/sql/operators/merge.py

```diff
@@ -50,8 +51,14 @@ def __init__(
         self.columns = columns or {}
         self.if_conflicts = if_conflicts
         task_id = task_id or get_unique_task_id("merge")
-
-        super().__init__(task_id=task_id, **kwargs)
+        super().__init__(
+            task_id=task_id,
+            **kwargs_with_datasets(
+                kwargs=kwargs,
+                input_ds=source_table,
```

Comment:
Shouldn't the `input_ds` be both `target_table` and `source_table` in this case?
It feels that `target_table` is also a `merge` input (it is a dataset expected to exist before calling this method).

Reply:
same comment as append

Reply:
In the case of `load_file`, it would only make sense adding it if the mode is "append" - but that is a good point